tests/test_relation.py

```diff
@@ -98,6 +101,25 @@ def test_insert_select(self):
             'real_id', 'date_of_birth', 'subject_notes', subject_id='subject_id+1000', species='"human"'))
         assert_equal(len(self.subject), 2*original_length)
 
+    @raises(InternalError)
```

Comment:
Let's catch the `InernalError` `insert` method and re-raise as `DataJointError` - in general, we don't want to unnecessarily expose underlying library to the end user.